.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Deploy ATAT API Infrastructure
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  deployInfrastructure:
+    name: Test STS
+    runs-on: ubuntu-latest
+    permissions:
+      # Required for the OIDC endpoint
+      id-token: write
+    steps:
+      # - name: Checkout code
+      #   uses: actions/checkout@v2
```

Comment:
I know we are testing this first with just `aws sts get-caller-identity`, and I assume we will use `actions/checkout@v2` when we want to actually deploy, but I wanted to make sure this was intentional.

Reply:
Yep! There's no need to check out the code (as-is) because we're not using it. I sort of wanted that present as a way to almost explicitly show that we're not doing that as part of the test today.